resources/qml/device-verification/Waiting.qml

```diff
@@ -21,9 +21,9 @@ Pane {
                 case "WaitingForOtherToAccept":
                     return qsTr("Waiting for other side to accept the verification request.");
                 case "WaitingForKeys":
-                    return qsTr("Waiting for other side to continue the verification request.");
+                    return qsTr("Waiting for other side to continue the verification.");
```

Comment:
I did not call it "verification request" anymore, because by this point the other user already accepted the request, so it isn't a request in my eyes anymore. Is that confusing?

Reply:
should we use 'process' in place of 'request'?  Like 'Waiting for the other side to continue the verification process...' or similar

Reply:
Yeah, that sounds a lot better.

Reply:
> I did not call it "verification request" anymore, because by this point the other user already accepted the request, so it isn't a request in my eyes anymore. Is that confusing?

Not at all, this is actually my point, and you did call it "verification request" (you probably read the diff backwards? x) )

I do agree with "process", though.

Reply:
You know, sometimes I can't read diffs. Yeah, totally on the same page here, thanks >.<